src/main/resources/i18n/messages.properties

```diff
@@ -1993,6 +1993,8 @@ SampleMetadataImportSelectFile.warning=Metadata uploaded will overwrite duplicat
 SampleMetadataImportSelectFile.dropzone=Click or drop Excel/CSV file containing metadata for samples in this project.
 SampleMetadataImportSelectFile.success={0} was read successfully.
 SampleMetadataImportSelectFile.error={0} failed to upload.
+SampleMetadataImportSelectFile.alert.valid.title=Validation Error
+SampleMetadataImportSelectFile.alert.valid.description=There are duplicate header names in file. Please correct the header names in the file and re-upload.
```

Comment:
Change this text to: `Duplicate header names were found in the uploaded file. Please correct the header names in the file and then re-upload.`

Reply:
Please see 85d4aa0732ecc77f2ab86d91404aec941445b095.